components/collection/CollectionCard.vue

```diff
@@ -24,34 +27,46 @@
 <script setup lang="ts">
 import { NeoSkeleton } from '@kodadot1/brick'
 import { CollectionWithMeta } from '@/components/rmrk/service/scheme'
+import type { TokenMetadata } from '@kodadot1/hyperdata'
+import CollectionDetail from './CollectionDetail.vue'
 import BasicImage from '@/components/shared/view/BasicImage.vue'
 import { processSingleMetadata } from '@/utils/cachingStrategy'
 import { sanitizeIpfsUrl } from '@/utils/ipfs'
-import CollectionDetail from './CollectionDetail.vue'
 
-import type { TokenMetadata } from '@kodadot1/hyperdata'
-
-const { urlPrefix } = usePrefix()
-const isLoadingMeta = ref(false)
-
-interface Props {
+const props = defineProps<{
   isLoading?: boolean
   collection?: CollectionWithMeta
-}
+  lazyLoading?: boolean
+}>()
 
-const props = defineProps<Props>()
+const isLoadingMeta = ref(false)
 const image = ref('')
 
-onMounted(async () => {
-  if (props.isLoading || !props.collection) {
-    return
-  }
+const { urlPrefix } = usePrefix()
 
+const getImageFromMetadata = async (collectionMetadata: string) => {
   isLoadingMeta.value = true
+
   const metadata = (await processSingleMetadata(
-    props.collection.metadata,
+    collectionMetadata,
   )) as TokenMetadata
+
   image.value = sanitizeIpfsUrl(getCollectionImage(metadata) || '')
   isLoadingMeta.value = false
+}
+
+onMounted(async () => {
+  if (props.isLoading || !props.collection) {
+    return
+  }
+
+  const meta = props.collection.meta
```

Comment:
update types `Property meta does not exist on type`

Reply:
done